docs/visualization/visualize_unit_metrics.ipynb

```diff
@@ -114,7 +114,7 @@
      "name": "stdout",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #13.    ax.plot(xaxis, data)
</pre><p>I think you need to show a good and a bad example here. Perhaps provide more text describing the difference. It is a bit light at the moment to understand. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AllenInstitute/openscope_databook/pull/158/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>